docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/snippets/basemanifest.bicep

```diff
@@ -0,0 +1,43 @@
+import radius as radius
+
+
+@description('Specifies the image of the container resource.')
+param magpieimage string = 'ghcr.io/radius-project/dev/magpiego:pr-498da3203f'
+
+@description('Specifies the port of the container resource.')
+param port int = 3000
```

Comment:
This seems weird to pass in as a parameter, as there's not really a scenario where this would change between environments. If we absolutely need a port number (see command below) let's hard code it